src/components/ec-submenu/ec-submenu.spec.js

```diff
@@ -0,0 +1,64 @@
+import { mount } from '@vue/test-utils';
```

Comment:
v-model is not tested, changing activeIndex should be tested too. Passing null to submenu should be tested. Passing empty array to the submenu prop should be tested.